### PR TITLE
Correcting installer forms

### PR DESCRIPTION
**Pull request by shefik**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Tests pass?       | yes
| Fixed tickets     | https://github.com/zikula/core/issues/2289
| Refs tickets      | https://github.com/zikula/core/pull/2399
| License           | MIT
| Doc PR            | -
| Changelog updated | no

### CODE REVIEW

**Comment by Guite** [comment]
Does this also relate to #2289 and/or #2290 ?

**Comment by shefik** [comment]
This only fixes https://github.com/zikula/core/issues/2289

File: src/lib/Zikula/Bundle/CoreInstallerBundle/Resources/views/structure.html.twig [hunk 1]
```diff
@@ -27,6 +27,13 @@
             $.webshims.activeLang('{{ lang }}');
             $.webshims.polyfill('forms');
         </script>
+        <script type="text/javascript">
+            (function($) {
+                $(document).ready(function() {
+                    $('label.control-label').removeClass('col-sm-2');
```
Comment (craigh) [inline src/lib/Zikula/Bundle/CoreInstallerBundle/Resources/views/structure.html.twig]: why is this needed?
